main.go

```
@@ -0,0 +1,62 @@
+package usdlregex
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"regexp"
+	"github.com/usdl-regex-go/model"
+)
+
+var stateRules model.StateRules
+
+/**
+ * Validate a drivers license
+ *
+ * @param stateCode string The state code for the drivers license
+ * @param licenseNumber string The drivers license number
+ *
+ * @return A boolean indicating if the drivers license is valid or any potential error found
+ *
+ */
+func ValidateDriversLicense(stateCode string, licenseNumber string) (match bool, err error) {
+	stateRulesFile, err := ioutil.ReadFile("stateRegex.json")
+	
+	if err != nil {
+		return
+	} 
+	
+	json.Unmarshal(stateRulesFile, &stateRules)
```

you're ignoring the error this could return, which is bad